index.html

```diff
@@ -0,0 +1,2110 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <!-- Required meta tags -->
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <!-- FontAwesome CDN -->
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.1/css/all.css" integrity="sha384-vp86vTRFVJgpjF9jiIGPEEqYqlDwgyBgEF109VFjmqGmIY/Y4HV4d3Gp2irVfcrp" crossorigin="anonymous" />
+  <!-- Bootstrap CSS -->
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1" crossorigin="anonymous" />
+  <link rel="stylesheet" href="asset/styles/style.css" />
+  <title>Newsweek</title>
+</head>
+
+<body>
+  <header>
+    <div class="header-main-container">
+      <nav>
+        <div class="row">
+          <div class="header-sub-container d-flex justify-content-between align-content-center">
+            <div class="header-logo d-none d-lg-flex col-7 justify-content-between align-items-center">
+              <div class="date-text">Wed, Jan 20, 2021</div>
+              <a href="#"><img class="header-logo-lg" src="asset/images/newsweek-logo.svg" alt="newsweek-logo" /></a>
```

Comment:
- [ ] The logo background color does not match the background color of the navbar, either use a svg or png image with transparent bg or fix the navbar color so it matches the one in the logo.
![Screenshot_2021-01-29 Newsweek(1)](https://user-images.githubusercontent.com/16193212/106256664-54f51c00-61fa-11eb-8e79-79eda746ff08.png)